scripts/install_environment.sh

```diff
@@ -90,8 +90,8 @@ function is_libvirtd_tcp_socket_enabled_and_running() {
 }
 
 function allow_libvirt_cross_network_traffic() {
-    # Flush LIBVIRT_FWI chain each time the network configuration is being
-    # updated by libvirt.
+    # Remove Reject rules from LIBVIRT_FWI and LIBVIRT_FWO chains each time the network
```

Comment:
capi test is passing again :partying_face: 
Now let's wait for the `e2e-metal-assisted-kube-api-late-binding-single-node` results
This removes the reject rules, it seems that when you drop the entire LIBVIRT_FWI chain this rule was blocking the traffic between minikube and the agent VMs
`-A LIBVIRT_FWO -i virbr1 -j REJECT --reject-with icmp-port-unreachable`
I guess that when we keep the `LIBVIRT_FWI` ACCEPT rules it would allow the traffic. Yet I figured there is no harm removing the reject rules on both chains